src/figuresClasses.ts

```diff
@@ -1,19 +1,110 @@
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
+
 export interface Figure {
+  color: Color,
+  shape: Shape,
 
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape;
+
+  constructor(public color: Color, public a: number, public b: number,
```

Comment:
Also consider refactoring this constructor because it looks pretty overloaded with code lines. You can create methods and move the logic there